packages/apps/tools/src/constants/kadena.ts

```diff
@@ -1,16 +1,15 @@
-import type { ChainwebNetworkId } from '@kadena/chainweb-node-client';
-
 import { env } from '@/utils/env';
 
-export type Network = Exclude<ChainwebNetworkId, 'development'>;
+export type DefinedNetwork = 'mainnet01' | 'testnet04';
+export type Network = DefinedNetwork | string;
 export type DevOption = 'BASIC' | 'BACKEND' | 'DAPP';
 
 type KadenaConstants = {
-  [K in Network]: {
+  [K in DefinedNetwork]: {
     label: string;
     API: string;
     apiHost: (params: { networkId: string; chainId: string }) => string;
-    estatsHost: () => string;
+    estatsHost?: () => string;
```

Comment:
Shouldn't this include the arguments?
```suggestion
    estatsHost?: ({ networkId: string, chainId: ChainId }) => string;
```

Why would this be optional? In the devnet you can reach it on `<host>/txs/account/k:554754f48b16df24b552f6832dda090642ed9658559fef9f3ee1bb4637ea7c94?limit=1000`

Reply:
estatsHost is now included as a prop and is not optional 👍